ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureClient.java

```diff
@@ -84,34 +55,35 @@ public void onResult(ThreeDSecureRequest request, ThreeDSecureLookup lookup, Exc
      *
      * @param activity Android FragmentActivity
      * @param request  the {@link ThreeDSecureRequest} with information used for authentication.
+     * @param callback {@link ThreeDSecureResultCallback}
      */
-    public void performVerification(final FragmentActivity activity, final ThreeDSecureRequest request, final ThreeDSecureLookupCallback callback) {
+    public void performLookup(final FragmentActivity activity, final ThreeDSecureRequest request, final ThreeDSecureResultCallback callback) {
```

Comment:
How come we're changing this from `performVerification` to `performLookup`? My understanding is the lookup is just one part of the 3DS verification process.